sdk/schemaregistry/schema-registry-avro/review/schema-registry-avro.api.md

```diff
@@ -0,0 +1,27 @@
+## API Report File for "@azure/schema-registry-avro"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { SchemaRegistry } from '@azure/schema-registry';
+
+// @public
+export class SchemaRegistryAvroSerializer {
+    constructor(registry: SchemaRegistry, schemaGroup: string, options?: SchemaRegistryAvroSerializerOptions);
+    autoRegisterSchemas: boolean;
+    deserialize(buffer: Buffer): Promise<any>;
+    readonly registry: SchemaRegistry;
```

Comment:
is it worthwhile having these be public?

Reply:
I guess not. I'll drop them.